pkg/block/fetcher.go

```diff
@@ -353,19 +472,16 @@ func (f *BaseFetcher) fetchMetadata(ctx context.Context) (interface{}, error) {
 	})
 
 	if err := eg.Wait(); err != nil {
-		return nil, errors.Wrap(err, "BaseFetcher: iter bucket")
+		return resp, errors.Wrap(err, "BaseFetcher: iter bucket")
```

Comment:
Could you explain me this change, please?

Reply:
`fetchMetadata` was returning `interface{}` previously, which could be nil, but now it returns `response` (not a pointer), so we need to return some value.

This change was done to make testing easier.